peep.c

```diff
@@ -3588,6 +3588,45 @@ Perl_rpeep(pTHX_ OP *o)
                 S_check_for_bool_cxt(o, 1, OPpTRUEBOOL, 0);
             /* FALLTHROUGH */
         case OP_COND_EXPR:
+            if (o->op_type == OP_COND_EXPR) {
+                /* Is there an empty "if" block or ternary true branch?
+                   If so, optimise away the OP_STUB if safe to do so. */
+                if (cLOGOP->op_other->op_type == OP_STUB &&
+                    ((cLOGOP->op_other->op_flags & OPf_WANT) != OPf_WANT_SCALAR)
+                ) {
+                    OP *stub = cLOGOP->op_other;
```

Comment:
move this line upwards 3-5 lines. I'd rather not see 3 x `cLOGOP->op_other` in a row even tho 90% chance the CC will dedupe all the derefs. clean/smaller ascii src code argument doesn't apply here b/c op_other is getting cached in an auto anyways. just cache it 3 lines up from where it is cached now.

Reply:
Ok, done.